native/src/components/__tests__/Feedback.spec.tsx

```diff
@@ -1,7 +1,8 @@
-import { fireEvent, render } from '@testing-library/react-native'
+import { fireEvent } from '@testing-library/react-native'
 import React from 'react'
 
 import buildConfig from '../../constants/buildConfig'
+import render from '../../testing/render'
```

Comment:
What's this about?

Reply:
Wraps the test automatically with `wrapWithTheme`. In web this does even more (like wrapping in a router), but for native this is all it does at the moment. So just a little helper.